README.md

```diff
@@ -42,25 +42,7 @@ Array of fields not to display. Overrides `colorInfo.fields`. If you only need t
 
 
 #### `colorInfo.languages`
-Defines which files and languages have color info enabled, and which color types are supported. The default language setting enables all css color values for `css`, `sass`, and `less` documents:
-
```

Comment:
This section is useful as it show users how they can add support for additional languages